### PR TITLE
Enable Draw

### PR DESCRIPTION
**Pull request by merttumer**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: loleaflet/src/control/Parts.js [hunk 1]
```diff
@@ -279,7 +279,7 @@ L.Map.include({
 		var docLayer = this._docLayer;
 
 		// At least for Impress, we should not fire this. It causes a circular reference.
-		if (this.getDocType() !== 'presentation') {
+		if (this.isPresentationOrDrawing()) {
```
Comment (tzolnai) [inline loleaflet/src/control/Parts.js:282]: Here you rewrote this incorrectly. It should be `if (!this.isPresentationOrDrawing()) {` I assume.

File: loleaflet/src/control/Parts.js [hunk 2]
```diff
@@ -298,14 +298,14 @@ L.Map.include({
...
-		if (this.getDocType() !== 'presentation') {
+		if (!this.isPresentationOrDrawing()) {
 			return;
 		}
 		this._socket.sendMessage('uno .uno:DuplicatePage');
 		var docLayer = this._docLayer;
 
 		// At least for Impress, we should not fire this. It causes a circular reference.
-		if (this.getDocType() !== 'presentation') {
+		if (this.isPresentationOrDrawing()) {
```
Comment (tzolnai) [inline loleaflet/src/control/Parts.js:308]: Here too.

File: loleaflet/src/control/Parts.js [hunk 3]
```diff
@@ -345,7 +345,7 @@ L.Map.include({
 		}
 
 		// At least for Impress, we should not fire this. It causes a circular reference.
-		if (this.getDocType() !== 'presentation') {
+		if (this.isPresentationOrDrawing()) {
```
Comment (tzolnai) [inline loleaflet/src/control/Parts.js:348]: And here.

**Review by tzolnai — CHANGES_REQUESTED** [review]
I marked some places, where I think the condition was changed incorrectly. I think that's why some cypress tests are failing on this PR. The test cases related to adding / removing slides are crashing because of this.

File: loleaflet/src/control/Parts.js (hunk 1 above)
Comment (kendy) [reply loleaflet/src/control/Parts.js]: @tzolnai: Thanks for that, @merttumer has fixed that in an additional commit.

**Comment by kendy** [comment]
I have no idea how to get rid of the "Changes requested", but the changes were done, so I'm just mering this.  @tzolnai , thanks again for your help!

**Review by tzolnai — APPROVED** [review]
Cypress tests pass now, so PR is OK for me.

**Pull request merged**